optional-container-engine/README.md

```diff
@@ -0,0 +1,76 @@
+# Deploy Bookshelf to Container Engine/Kubernetes
+
+This optional tutorial will walk you through how to deploy the Bookshelf sample application to [Google Container Engine](https://cloud.google.com/container-engine/). This tutorial is also applicable to [Kubernetes](http://kubernetes.io/) outside of Container Engine, but may require additional steps for external load balancing.
+
+## Pre-requisites
+
+1. Create a project in the [Google Cloud Platform Console](https://console.cloud.google.com).
+
+2. [Enable billing](https://console.cloud.google.com/project/_/settings) for your project.
+
+3. [Enable APIs](https://console.cloud.google.com/flows/enableapi?apiid=datastore,pubsub,storage_api,logging,plus) for your project. The provided link will enable all necessary APIs, but if you wish to do so manually you will need Datastore, Pub/Sub, Storage, and Logging.
+
+4. Install the [Google Cloud SDK](https://cloud.google.com/sdk)
+
+        $ curl https://sdk.cloud.google.com | bash 
+        $ gcloud init
+
+5. Install [Docker](https://www.docker.com/).
+
+## Create a cluster
+
+Create a cluster for the bookshelf application:
+
+    gcloud container clusters create bookshelf \
+        --scope "https://www.googleapis.com/auth/userinfo.email","cloud-platform"
```

Comment:
Wondering if we should specify num_instances, since default is 3, which means 3x as expensive as if we specified 1. Even 2 would be cheaper and demonstrate the cross-node networking stuff.